src/Data/Reader/EntityReader.php

```diff
@@ -226,4 +226,19 @@ private function resetCountCache(): void
         }
         $this->countCache = new CachedCount($newQuery);
     }
+
+    private function normalizeCriteria(array $criteria): array
+    {
+        foreach ($criteria as $field => $direction) {
+            if (is_int($direction)) {
+                match ($direction) {
+                    SORT_DESC => $direction = 'DESC',
+                    default => $direction = 'ASC',
```

Comment:
are you sure the `$direction` var can not contain other values like `"asc"` or `"desc"`?

Reply:
Yes, checked for int in above.